incorrect number of tests in report output when test was skipped

I prepared a test suite like:

``` javascript
var assert = require("assert");

describe("this is", function() {
  it("test1", function() {
    assert.equal(1, 1);
  });

  it("test2", function() {
    assert.equal(1, 2);
  });

  it.skip("test3", function() {
    assert.equal(1, 3);
  });
});
```

and run mocha with mocha-tap-reporter

``` bash
$ mocha --reporter mocha-tap-reporter mytest.js
```

I assume it should return the same result output as described [here](https://github.com/lyroyce/mocha-tap-reporter/blob/master/README.md)

But  unfortunately it is returning a little bit different result as you can see below

```
1..3
ok 1 this is test1
not ok 2 this is test2
#  AssertionError: 1 == 2
#      at Context.<anonymous> (/Volumes/Data/workspace/tests/test2.js:10:10)
#      at Test.Runnable.run (/Volumes/Data/workspace/tests/node_modules/mocha/lib/runnable.js:196:15)
#      at Runner.runTest (/Volumes/Data/workspace/tests/node_modules/mocha/lib/runner.js:373:10)
ok 3 this is test3 # SKIP -
# tests 2
# pass 1
# fail 1
# skip 1
```

this part:
`# tests 2` 
is incorrect because there are 3 tests in total.

--- a/lib/etap.js
+++ b/lib/etap.js
@@ -59,7 +59,7 @@ function eTAP(runner) {
   });
 
   runner.on('end', function(){
-    console.log('# tests ' + (passes + failures));
+    console.log('# tests ' + (passes + failures + skips));
     console.log('# pass ' + passes);
     console.log('# fail ' + failures);
     console.log('# skip ' + skips);

